gammapy/maps/regionnd.py

```diff
@@ -154,6 +155,37 @@ def plot_region(self, ax=None, **kwargs):
         ax = self.geom.plot_region(ax, **kwargs)
         return ax
 
+    def plot_mask(self, ax=None, color="k", label=""):
+        """Plot region
+
+        Parameters
+        ----------
+        ax : `~astropy.vizualisation.WCSAxes`
+            Axes to plot on. If no axes are given,
+            the region is shown using the minimal
```

Comment:
This is a copy and paste error: here the ax is not a `WCSAxes` and there is no region plotting, but only a spectrum...

Reply:
Ahh right, my fault! I've corrected it.